paddle/phi/api/yaml/op_compat.yaml

```
@@ -3799,6 +3799,12 @@
   inputs :
     x : X
 
+- op: prune_gate_by_capacity
+  inputs:
+    {gate_idx: GateIdx, expert_count: ExpertCount, n_expert: n_expert, n_worker: n_worker}
```

```suggestion
    {gate_idx: GateIdx, expert_count: ExpertCount,}
```
这里主要是为了配置名称的映射关系。默认是名称相同的，所以如果同名的话，就不用添加在yaml中，更建议删去。